projects/dotcms-field-elements/src/components/dot-date-range/dot-date-range.tsx

```diff
@@ -119,8 +119,8 @@ export class DotDateRangeComponent {
 
     @Watch('value')
```

Comment:
Can we add to the description an example, like:

```Value formatted with start and end date splitted with a comma, ex: 02-20-2012,01-10-2012```

The same should go for any other prop that have like a very specific format of date and stuff, cc: @hmoreras